docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,500 +1,594 @@
-seamlessly connect to resources in another cluster, the **leaf cluster**, while
-remaining authenticated with only a single Auth Service. The leaf cluster can
-run behind a firewall without any ingress ports open.
+After you establish a trust relationship between the root and leaf clusters, the leaf cluster can
+run behind a firewall without any inbound ports open. The leaf cluster creates an outbound reverse 
+SSH tunnel to the root cluster and keeps the tunnel open.
+When a user tries to connect to any resource in the leaf cluster, the connection goes through the 
+reverse tunnel using the root cluster's proxy.
 
-Uses for Trusted Clusters include:
+## Who uses trusted clusters?
+
+Most organizations don't need to configure trusted clusters. In most cases, you can enroll
+resources and add reverse-tunnel proxies without creating a new cluster. Typically, a single 
+teleport cluster can scale to hundreds of thousands of connected resources.
+
+However, there are a few specific scenarios where trusted clusters can be paricularly useful.
+For example, if you have a large and widely-distributed infrastructure or must provide access
+to resources for external agencies, contractors, or clients, you might benefit from setting up 
+a trusted cluster. The most common use cases for trusted clusters include:
 
 - Managed service providers (MSP) remotely managing the infrastructure of their clients.
 - Device manufacturers remotely maintaining computing appliances deployed on premises.
 - Large cloud software vendors managing multiple data centers using a common proxy.
 
-Here is an example of an MSP using Trusted Clusters to obtain access to client clusters:
-![MSP Example](../../../img/trusted-clusters/trusted-clusters@1.5x.svg)
-
-A leaf cluster always creates an outbound reverse SSH tunnel to the root cluster and keeps the tunnel open.
-
-<Notice type="tip">
-Individual nodes and proxies can create reverse tunnels to proxy services without creating a new cluster.
-You don't need to set up a trusted cluster just to connect a couple of servers, kubernetes clusters or
-databases behind a firewall. For more information, see how to [join a service to your cluster](../../agents/join-services-to-your-cluster.mdx).
-</Notice>
-
-When a user tries to connect to any resource inside the leaf cluster using the root's proxy
-the connection goes through the reverse tunnel.
-
-This guide will explain how to:
-
-- Add and remove Trusted Clusters using CLI commands.
-- Enable/disable trust between clusters.
+## How it works
+
+In the following example, a managed service provider uses three independent clusters to 
+provide access to clients in different regions:
+
+- Cluster `sso.example.com` is the root cluster. This cluster might have its own resources or be 
+strictly used for collecting audit logs and authenticating users.
+- Clusters `us-east-1a` and `us-east-1b` are two independent clusters in different availability zones.
```

Comment:
Similarly, how about leaf1.example.com and leaf2.example.com?

I'm afraid that by using the name of AWS availability zones, we might be suggesting to readers that trusted clusters are a way to get higher availability. (which is not actually the case)

This change would also address an inconsistency - right now we're using a fully qualified domain name to refer to the root cluster but not for the leaf clusters.